corda/corda
# Replace 'by' with 'using' to avoid overloading Kotlin keyword

The `ContractsDSL` overloads the Kotlin `by` keyword. User feedback from training courses indicates that a better choice would be `using` to avoid any confusion with built-in DSLs.

This PR covers

* changing `by` to `using` in `ContractsDSL`
* refactoring all uses within the Corda codebase
* providing a `ReplaceWith` definition to assist downstream users with automatic migration

To test:

1. Open `CommercialPaper` in Intellij and locate the `using` keyword (line 146)
2. Change to `by`
3. Observe deprecated status with comment
4. Use auto-correct to replace with `using` either at individual or project level
